### PR TITLE
[NGS-1207] Yeahmobi integration

### PR DESCRIPTION
**Pull request by ChillBroYo**
## Context

Initial for Yeahmobi

 ## Jira Link

Jira: https://tapjoy.atlassian.net/browse/NGS-1207

## Related PRs
- tpe_prebid: https://github.com/Tapjoy/tpe_prebid_service/pull/182
- go-prebid: https://github.com/Tapjoy/go-prebid/pull/112
- mediation_bid_service: https://github.com/Tapjoy/mediation_bid_service/pull/1619
- Chef: https://github.com/Tapjoy/5rocks-chef/pull/3257
- ad_service: https://github.com/Tapjoy/ad_service/pull/1196

### CODE REVIEW

**Review by umutdemirel — COMMENTED** [review]
Looks good with some comments, I assume all other DSP details are in place because they've already had adapter integration. You can also check Mintegral integration to see those files: https://github.com/Tapjoy/tpe_prebid_service/pull/169/files (mintegral.yaml, mintegral.json, config.go updates etc.)

So we just need to test locally and see the generated request and if end to end flow works fine. I'll approve when the test is done.

File: adapters/tjx_yeahmobi/yeahmobi.go [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+type yeahmobiUserExt struct {
+	TagID    string `json:"tagid"`
+	Region   string `json:"region"`
+	Language string `json:"language"`
+}
+
+type yeahmobiBannerExt struct {
+	HasPrivate  int    `json:"hasPrivate"`
+	AllowAdType string `json:"allow_ad_type"`
+}
```
[the remark this answers is not in the document]
Comment (ChillBroYo) [reply adapters/tjx_yeahmobi/yeahmobi.go:61]: They appear to be, I've never seen them in any other dsp

File: adapters/tjx_yeahmobi/yeahmobi.go [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+	headers.Add("Content-Type", "application/json;charset=utf-8")
+	headers.Add("Accept", "application/json")
+	headers.Add("User-Agent", "prebid-server/1.0")
+
+	errs := make([]error, 0, len(request.Imp))
+	var err error
+
+	yeahmobiRequest := *request
+
+	// yeahmobi does not have a app extension
```
[the remark this answers is not in the document]
Comment (ChillBroYo) [reply adapters/tjx_yeahmobi/yeahmobi.go:90]: I don't see anything they are passing and I'm not sure what we would be passing

**Review by Rajat2809 — COMMENTED** [review]
Missing Config change for the endpoint. I think you need to modify the endpoint.
For reference:
https://github.com/Tapjoy/tpe_prebid_service/blob/9e6765a56b64ba68aebd6d3f27f319787a5ae24f/config/config.go#L1073-L1074


Once the local testing is successful I can approve this or @umutdemirel Can do it.

**Review by Rajat2809 — APPROVED** [review]
LGTM. Tested it locally worked

**Pull request merged**